backend/src/packages/auth/auth.service.ts

```diff
@@ -1,25 +1,32 @@
 import {
   type UserSignInRequestDto,
+  type UserSignInResponseDto,
   type UserSignUpRequestDto,
   type UserSignUpResponseDto,
 } from '~/packages/users/libs/types/types.js';
 import { type UserService } from '~/packages/users/user.service.js';
 
+import { createToken } from './libs/helpers/token/create-token/create.token.helper.js';
+
 class AuthService {
   private userService: UserService;
 
   public constructor(userService: UserService) {
     this.userService = userService;
   }
 
-  public signIn(userRequestDto: UserSignInRequestDto): UserSignInRequestDto {
-    return userRequestDto;
+  public signIn(userRequestDto: UserSignInRequestDto): UserSignInResponseDto {
+    const id = Math.floor(Math.random() * 100); //replase with proper id from database
```

Comment:
you should to do it here

Reply:
You mean implement user find in repositiry&